nifi-nar-bundles/nifi-standard-services/nifi-hbase_1_1_2-client-service-bundle/nifi-hbase_1_1_2-client-service/pom.xml

```diff
@@ -25,7 +25,7 @@
     <artifactId>nifi-hbase_1_1_2-client-service</artifactId>
     <packaging>jar</packaging>
     <properties>
-        <hbase.version>1.1.13</hbase.version>
+        <hbase.version>1.4.13</hbase.version>
```

Comment:
I think we had discussion about this before, but since the service is named 1_1_2, are we ok going to 1.4.13? I think last time we decided to go to the latest 1.1.x release to limit the changes.

Reply:
Oh my bad, I don't recall about this discussion. I discussed with some HBase folks and changes are non existent on the client side when it comes to data push/pull across the 1.x lines (even between 1.x and 2.x to be honest, changes are mostly around administration and management APIs). This upgrade is motivated by transitive dependencies but I can remove this specific change if needed.